### DISCUSSION TITLE
Bind for resizeactive not working, but it does work via hyprctl dispatch

### DISCUSSION THREAD

**Discussion by jennydaman** [hyprwm/Hyprland#5506] [Q&A]
I am trying to create some binds for resizing windows like this;

```ini
binde = $mainMod SHIFT, L, resizeactive,  50   0
```

Nothing happens, Yet it seems to work if I pass it through `hyprctl dispatch`:

```ini
binde = $mainMod SHIFT, L, exec, hyprctl dispatch resizeactive  50   0
```

Could this be a bug, or is something wrong with my config?

Hyprland version 0.37.1 on NixOS unstable, Intel Integrated Graphics.

**Comment by vaxerski** [comment]
I think this was fixed in git

**Reply by jennydaman** [reply]
Great! so I just wait for the next release to be available?
